raiden/ui/cli.py

```diff
@@ -605,6 +605,12 @@ def run(ctx: Context, **kwargs: Any) -> None:
             profiler.stop()
 
 
+# List of available options, use by the scenario player
+FLAG_OPTIONS = {param.name.replace("_", "-") for param in run.params if param.is_flag}
+FLAG_OPTIONS = FLAG_OPTIONS.union({"no-" + opt for opt in FLAG_OPTIONS})
+KNOWN_OPTIONS = {param.name.replace("_", "-") for param in run.params}.union(FLAG_OPTIONS)
```

Comment:
@nlsdfnbch are these sufficient for what you need?

Reply:
It works for me, see https://github.com/raiden-network/scenario-player/pull/375, so at least the functionality is there. @nlsdfnbch still might disagree with the style or details.